extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -464,7 +464,7 @@ export class AmpConsent extends AMP.BaseElement {
         const keys = Object.keys(config['policy']);
         for (let i = 0; i < keys.length; i++) {
           if (keys[i] != 'default') {
-            user().warn(TAG, `policy ${keys[i]} is currently not supported` +
+            user().warn(TAG, `policy ${keys[i]} is currently not supported ` +
```

Comment:
Can we revert the change here.

Reply:
Sure, can I ask what the issue is? Currently there's no space between "supported" and "and".

Reply:
😄 my bad. nvm..